rpcs3/Crypto/utils.h

```diff
@@ -6,7 +6,7 @@
 
 #include "../../Utilities/types.h"
 
-#define MAX_PATH 4096
+#define EMU_MAX_PATH 4096

```

Comment:
If you are already at it, use enum instead. and maybe name it CRYPO_PATH_SIZE.

Reply:
Preference for 'enum' over 'const int'?

Reply:
yes.